nifi-extension-bundles/nifi-standard-services/nifi-schema-registry-service-api/src/main/java/org/apache/nifi/schemaregistry/services/MessageNameResolver.java

```diff
@@ -32,7 +29,7 @@
  * message processing and routing based on the resolved message type.
  * </p>
  */
-public interface MessageNameResolver extends ControllerService {
+public interface MessageNameResolver  {
```

Comment:
Extra space should be removed:
```suggestion
public interface MessageNameResolver {
```

Reply:
Restored previous inheritance.